src/Test.FunctionalTests.Sarif/default.configuration.xml

```diff
@@ -12,4 +12,7 @@ This file contains the default rule configurations for the functional tests of t
   <Properties Key="SARIF2007.ExpressPathsRelativeToRepoRoot.Options">
     <Property Key="RuleEnabled" Value="Disabled" />
   </Properties>
+  <Properties Key="SARIF2016.FileUrisShouldBeRelative.Options">
+    <Property Key="RuleEnabled" Value="Disabled" />
```

Comment:
>Disabled [](start = 39, length = 8)

Why is this rule disabled by default? I am beginning to think we've made a mistake: these rules should be notes, enabled by default. The validation tool should not show them unless appropriate verbosity is set.